README.markdown

```diff
@@ -85,9 +85,10 @@ server {
                 return
             end
 
-            local answers, err = r:query("www.google.com")
+            local answers, err, tries = r:query("www.google.com", nil, {})
             if not answers then
                 ngx.say("failed to query the DNS server: ", err)
+                ngx.say("retry historie: \n  "..table.concat(tries, "\n  "))
```

Comment:
Better replace `..` with `,` on this line since this Lua concatenation operation is not necessary on the Lua land.

Also, BTW, we always need spaces around infix operators like `..`.

Reply:
Also, better avoid the space before `\n`, which would become a line trailing space in the output.